devtools/conda-envs/environment.yml

```diff
@@ -4,16 +4,19 @@ channels:
   - omnia
   - rdkit
   - samoturk
+  - schrodinger
 dependencies:
   - pytest
   - biopandas
   - jupyter
+  - wandb
   - matplotlib
   - numpy
-  - pandas
-  - rdkit
+  - pandas= 2.1
+  - rdkit=2021.09
   - parmed  
   - pmw
+  - pymol-bundle 
   - pymol
```

Comment:
Why are there two pymol packages here? `pymol-bundle` should be enough, or do we need both?